src/lib.rs

```diff
@@ -58,18 +58,23 @@
 #![cfg_attr(not(test), no_std)]
 //  Nightly features
 #![cfg_attr(feature = "compile-time-ratio", allow(incomplete_features))]
+// https://github.com/rust-lang/rust/issues/76560
 #![cfg_attr(feature = "compile-time-ratio", feature(generic_const_exprs))]
+// https://github.com/rust-lang/rust/issues/79024
 #![cfg_attr(feature = "nightly-async-iterator", feature(async_iterator))]
+// https://github.com/rust-lang/rust/issues/18598
 #![cfg_attr(feature = "nightly-coerce-unsized", feature(coerce_unsized))]
-#![cfg_attr(feature = "nightly-dispatch-from-dyn", feature(dispatch_from_dyn))]
 #![cfg_attr(
     any(
         feature = "nightly-dispatch-from-dyn",
         feature = "nightly-coerce-unsized"
     ),
     feature(unsize)
 )]
-#![cfg_attr(feature = "nightly-generator-trait", feature(generator_trait))]
+// https://github.com/rust-lang/rust/issues/43122
+#![cfg_attr(feature = "nightly-generator-trait", feature(generator_trait))] // Unused ????
+#![cfg_attr(feature = "nightly-dispatch-from-dyn", feature(dispatch_from_dyn))] // Unused ????
```

Comment:
I see you added this Unused comment, if they are, maybe they can be removed?

Reply:
I'll look into the git blame to see why it was originally added :smile:

Reply:
They are certainly used - I'm not sure what I missed when I added those comments.

Reply:
I do believe they are used. Specifically, I do believe that users on nightly can "activate" them by compile the `static-rc` crate with the `nightly-generator-trait` and `nightly-dispatch-from-dyn` features.

I would recommend removing the added comments, and leaving them as is.